scripts/fast-reboot-dump.py

```diff
@@ -271,9 +285,9 @@ def main():
         print("Target directory '%s' not found" % root_dir)
         return 3
     all_available_macs, map_mac_ip_per_vlan = generate_fdb_entries(root_dir + '/fdb.json')
-    arp_entries = generate_arp_entries(root_dir + '/arp.json', all_available_macs)
+    neighbor_entries = generate_neighbor_entries(root_dir + '/arp.json', all_available_macs)
     generate_default_route_entries(root_dir + '/default_routes.json')
-    garp_send(arp_entries, map_mac_ip_per_vlan)
+    garp_ndp_send(neighbor_entries, map_mac_ip_per_vlan)
```

Comment:
While you are at it, suggest name change to `send_garp_nd` since method naming convention start by verb.